plugins/vln/README_VLN.md

```diff
@@ -6,7 +6,7 @@ Current stable versions for VLAN -
  - beta: v1beta.fukuoka.1724
```

Comment:
could you refer to the link(s) for downloading the artifact(s)?

Reply:
@ayasuda2003 .. Added the links for pointing to corresponding versions